### PR TITLE
add as per issue #88

### PR DESCRIPTION
**Pull request by rajarshig**
Used [Release notes](https://github.com/camilamaia/scanapi/releases) & [keepachangelog.com](https://keepachangelog.com/en/1.0.0/) reference.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+### Added
+- Used env variables from os
+
+## [0.0.11] - 2019-08-09
+### Added
+- Added Docker file
+
+## [0.0.10] - 2019-08-09
+### Added
+- Fix vars interpolation
```
Comment (camilamaia) [inline CHANGELOG.md]: Maybe this could be also under `Fixed`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### Removed
+- Removed requirements files and put every dependency under setup.py
+- Removed dcvars key
+
+## [0.0.14] - 2019-10-09
+### Changed
+- Bumped version
+
+## [0.0.12] - 2019-08-15
```
Comment (camilamaia) [inline CHANGELOG.md]:
```suggestion
## [0.0.12] - 2019-08-14
```

Comment (camilamaia) [inline CHANGELOG.md]:
```suggestion
## [0.0.12] - 2019-08-14
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- Fixed join of urls to keep the last slash
+- Used dot notation to access responses inside api spec
+- Renamed option report_path to output_path
+- Reporter option -r, --reporter [console|markdown|html]
+
+### Removed
+- Removed requirements files and put every dependency under setup.py
+- Removed dcvars key
+
+## [0.0.14] - 2019-10-09
```
Comment (camilamaia) [inline CHANGELOG.md]:
```suggestion
## [0.0.13] - 2019-10-07
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- Increased coverage
+- Fixed join of urls to keep the last slash
+- Used dot notation to access responses inside api spec
+- Renamed option report_path to output_path
+- Reporter option -r, --reporter [console|markdown|html]
+
+### Removed
+- Removed requirements files and put every dependency under setup.py
+- Removed dcvars key
+
```
Comment (camilamaia) [inline CHANGELOG.md]: I think release v0.0.14 is missing here: https://github.com/camilamaia/scanapi/releases/tag/v0.0.14

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+## [Unreleased]
+
+## [0.0.17] - 2019-12-19
+### Added
+- Added PyPI Test section to CONTRIBUTING.md
+- Added templates to pypi package - fix [#84](https://github.com/camilamaia/scanapi/issues/84)
+
+
+## [0.0.16] - 2019-12-18
+### Changed
```
Comment (camilamaia) [inline CHANGELOG.md]:
```suggestion
### Fixed
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.17] - 2019-12-19
```
Comment (camilamaia) [inline CHANGELOG.md]:
```suggestion
## [0.0.17] - 2019-12-18
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+## [0.0.15] - 2019-12-14
+## Added
+- CodeCov Setup
+- CircleCI Setup
+
+### Changed
+- Updated Documentation
+- Increased coverage
+- Fixed join of urls to keep the last slash
```
Comment (camilamaia) [inline CHANGELOG.md]: Maybe this line could be under the `Fixed` section instead of `Added`. What do you think?

File: CHANGELOG.md (hunk 3 above)
Comment (rajarshig) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 7 above)
Comment (rajarshig) [reply CHANGELOG.md]: Right, updated.

**Comment by rajarshig** [comment]
@camilamaia thanks a lot for the review! Changes have been updated in 
commit [https://github.com/camilamaia/scanapi/pull/90/commits/29c5575a0c9a2c0600a97e98a9b65a8ddb21c2ac](url). Let me know your thoughts.

**Comment by camilamaia** [comment]
@rajarshig it is good to go, let's merge it! 🚀

**Pull request merged**